modules/integration/tests-common/clients/admin/src/main/resources/admin-api.yaml

```diff
@@ -108,6 +108,49 @@ info:
 servers:
   - url: https://apis.wso2.com/api/am/admin/v3
 paths:
+  ######################################################
+  # The "Content Search Results" Throttling Policies
+  ######################################################
+  /throttling/policies/search:
+    get:
+      tags:
+        - Unified Search
+      summary: |
+        Retrieve/Search Throttling Polcies
+      description: |
+        This operation provides you a list of available Throttling Policies qualifying the given keyword match.
+      parameters:
+        - name: query
+          in: query
+          description: |
+            **Search**.
+            You can search by providing a keyword. Allowed to search by type only.
+          schema:
+            type: string
+      responses:
+        200:
+          description: |
+            OK.
+            List of qualifying Throttling Policies is returned.
+          headers:
+            Content-Type:
+              description: The content type of the body.
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ThrottlePolicyDetailsList'
+      security:
+        - OAuth2Security:
+            - apim:admin
+            - apim:tier_view
+      x-code-samples:
+        - lang: Curl
+          source: 'curl -k -H "Authorization: Bearer ae4eae22-3f65-387b-a171-d37eaa366fa8"
+          "https://127.0.0.1:9443/api/am/admin/v3/search?query=type:sub"'
```

Comment:
Is this path correct?

Reply:
Fixed with 3106d801961d2d8e82a5a35117427b8337a92455